website/src/theme/ReactLiveScope/index.ts

```diff
@@ -6,13 +6,17 @@
  */
 
 import React from 'react';
-import * as components from './components';
+import {createRequire} from 'react-runner';
```

Comment:
Another option is ex-exporting it from '@theme-original/CodeBlock`, I tried but got error, seems it only support default export